node/collation-generation/src/lib.rs

```diff
@@ -273,6 +280,8 @@ async fn handle_new_activations<Context: SubsystemContext>(
 					},
 				};
 
+				metrics.on_collation_generated();
```

Comment:
this check this is the right place for this

Reply:
yep, that seems right.